src/components/Match/Overview/Timeline.jsx

```diff
@@ -165,38 +167,16 @@ export default ({ match }) => {
                           <section>
                             {match.players
                               .filter(player => player.player_slot === obj.player_slot)
-                              .map(player => (
-                                <aside style={{ color: playerColors[obj.player_slot] }}>
-                                  <img
-                                    src={heroes[player.hero_id]
-                                      ? `${API_HOST}/apps/dota2/images/heroes/${getShortHeroName(heroes[player.hero_id].name)}_icon.png`
-                                      : '/assets/images/blank-1x1.gif'
-                                    }
-                                    role="presentation"
-                                  />
-                                  {player.name || player.personaname || strings.general_anonymous}
-                                </aside>
-                              ))
+                              .map(player => <PlayerThumb {...player} />)
                             }
                             <span>
                               {obj.key ? strings.timeline_firstblood_key : strings.timeline_firstblood}
                             </span>
                             {obj.key &&
-                              <aside>
-                                <img
-                                  src={`${API_HOST}/apps/dota2/images/heroes/${getShortHeroName(obj.key)}_icon.png`}
-                                  role="presentation"
-                                />
-                                {match.players
-                                  .filter(player =>
-                                    player.hero_id === heroes[Object.keys(heroes).filter(key => heroes[key].name === obj.key)].id,
-                                  ).map(player => (
-                                    <div style={{ color: playerColors[player.player_slot] }}>
-                                      {player.name || player.personaname || strings.general_anonymous}
-                                    </div>
-                                  ))
-                                }
-                              </aside>
+                              <PlayerThumb
+                                {...match.players.find(player =>
+                                  player.hero_id === heroesArr('find')(hero => hero.name === obj.key).id)}
```

Comment:
This is existing behavior, I just factored it out into a component. I think it selects a 'blank' icon for that hero that presumably takes up the same space or something.